app/views/users/show.html.erb

```diff
@@ -10,21 +14,13 @@
 </section>
 
 <section>
-  <h2>Recent Posts</h2>
   <div>
-    <h3>Post 1</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-  </div>
-  <div>
-    <h3>Post 2</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
-  </div>
-  <div>
-    <h3>Post 3</h3>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus dolores maxime quos, nam mollitia
-    aliquid iste at porro sunt consequatur.</p>
+    <% @user.posts.each do |post| %>
+      <h3>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_posts` method in the `user` model to render only the last 3 recent posts.